howso/residuals.amlg

```diff
@@ -750,11 +790,21 @@
 												(if focal_case
 													(query_not_in_entity_list (list case_id focal_case))
 													(query_not_in_entity_list (list case_id))
-												)
+												)									
 												(if ignore_null_action_feature
 													(query_not_equals feature (null))
 													(list)
 												)
+												;if both action and context filter queries are provided while 'case_ids' were not provided at the beginning of the function, 
+												;only query the cases specifed by the context filter query, effectively holding out the action set. Otherwise query everything.
+												(if (and 
+														(not provided_case_id_flag)
+														(!= (size action_condition_filter_query) 0)
+														(!= (size context_condition_filter_query) 0)
+													)
+													context_condition_filter_query
```

Comment:
I don't understand why this should only go here when `action_condition_filter_query` is also populated? It seems that with these new parameters we should be able to see the accuracy across any case of the model (no action_condition) using only a subset of the cases (non-empty context_condition), but this seems to suggest that this would not be the case. Is this intentional?

Reply:
Yea, this was intentional, didn't seem like it was useful to see the entire of the entire model using a subset of the model. Certainly debatable